datasets/pass/README.md

```diff
@@ -111,7 +111,7 @@ A data point comprises an image and its meta-data:
 
 ### Data Splits
 
-All the data is contained in training set. The training set has 1.4M (1,439,719) instances.
+All the data is contained in the training set. The training set has 1,439,588 instances as this implementation corresponds to the most recent release from the [version history](https://github.com/yukimasano/PASS/blob/main/version_history.txt).
```

Comment:
I'd add the version, in case they add another version soon and this paragraph becomes outdated

```suggestion
All the data is contained in the training set. The training set has 1,439,588 instances as this implementation corresponds to the most recent release (v3) from the [version history](https://github.com/yukimasano/PASS/blob/main/version_history.txt).
```